### ISSUE TITLE
[Graphql] Update node version

### ISSUE THREAD

**Issue by lmacielvieira** [metaspace2020/metaspace#1318]
Update node.js version:

- [x] docker
- [x] staging
- [ ] production

**Comment by sergii-mamedov** [comment]
JFYI: https://github.com/metaspace2020/metaspace/pull/637

**Comment by lmacielvieira** [comment]
### Install NVM (already installed)
```
curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash 
source ~/.bashrc   
```

### Upgrade to the new version (v16.20)

v18 is the LTS, but it is not compatible with ubuntu 18.04.

```
# Install the version
nvm install 16.20.0

# Set 16.20.0 as default
nvm alias default 16.20.0

# Update default version to be used
nvm use default

```

**Comment by sergii-mamedov** [comment]
I see next on logs:
```
npm notice 
npm notice New major version of npm available! 8.19.4 -> 9.6.6
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.6
npm notice Run npm install -g npm@9.6.6 to update!
npm notice 
```
I think it is necessary to update npm too?

**Comment by lmacielvieira** [comment]
> I see next on logs:
> 
> ```
> npm notice 
> npm notice New major version of npm available! 8.19.4 -> 9.6.6
> npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.6.6
> npm notice Run npm install -g npm@9.6.6 to update!
> npm notice 
> ```
> 
> I think it is necessary to update npm too? @lmacielvieira

For now let's leave this version, which is matching to node v16. Once we update to the latest node, after updating ubuntu, we also update the npm.
